pennylane/ops/op_math/controlled_ops.py

```diff
@@ -117,68 +123,75 @@ class ControlledQubitUnitary(ControlledOp):
     def _unflatten(cls, data, metadata):
-        return cls(
-            data[0], control_wires=metadata[0], control_values=metadata[1], work_wires=metadata[2]
-        )
+        return cls(data[0], metadata[0], control_values=metadata[1], work_wires=metadata[2])
 
     # pylint: disable=arguments-differ, too-many-arguments, unused-argument, too-many-positional-arguments
     @classmethod
     def _primitive_bind_call(
         cls,
         base,
-        control_wires: WiresLike,
-        wires: WiresLike = (),
+        wires: WiresLike,
+        control_wires: WiresLike = "unset",
         control_values=None,
         unitary_check=False,
         work_wires: WiresLike = (),
     ):
-        wires = Wires(() if wires is None else wires)
+        cls._deprecate_control_wires(control_wires)
+
         work_wires = Wires(() if work_wires is None else work_wires)
 
         if hasattr(base, "wires") and len(wires) != 0:
             warnings.warn(
                 "base operator already has wires; values specified through wires kwarg will be ignored."
             )
-            wires = Wires(())
+            # wires = Wires(())
 
-        all_wires = control_wires + wires
         return cls._primitive.bind(
-            base, control_wires=all_wires, control_values=control_values, work_wires=work_wires
+            base, wires=wires, control_values=control_values, work_wires=work_wires
         )
 
+    @classmethod
+    def _deprecate_control_wires(cls, control_wires):
+        if control_wires != "unset":
+            warnings.warn(
+                "The control_wires input to ControlledQubitUnitary is deprecated and will be removed in v0.42. "
+                "Instead, please use wires, following wires=controlled_wires+target_wire.",
+                qml.PennyLaneDeprecationWarning,
+            )
+
     # pylint: disable=too-many-arguments,too-many-positional-arguments
     def __init__(
         self,
         base,
-        control_wires: WiresLike,
-        wires: WiresLike = (),
+        wires: WiresLike,
+        control_wires: WiresLike = "unset",
         control_values=None,
         unitary_check=False,
         work_wires: WiresLike = (),
     ):
-        wires = Wires(() if wires is None else wires)
+        self._deprecate_control_wires(control_wires)
+
+        if wires is None or len(wires) == 0:
+            raise TypeError("Must specify a set of wires. None is not a valid `wires` label.")
         work_wires = Wires(() if work_wires is None else work_wires)
-        control_wires = Wires(control_wires)
+        control_wires = wires[:-1]  # default
 
         if hasattr(base, "wires") and len(wires) != 0:
             warnings.warn(
                 "base operator already has wires; values specified through wires kwarg will be ignored."
             )
-            wires = Wires(())
+            # base should have the target_wires. Then control_wires should be the ones in wires that are not in base.wires
```

Comment:
I feel that specifying wires with `base` being an operator is going to be ambiguous regardless of what we do. I believe we're deprecating that soon as well?

Reply:
Indeed. Since this PR is altering way more than the other I did this first. Anything comes here is temprory and handwaving just  to pass existing usage.
My plan for deprecating Operator input is to take the matrix and wires out from the op along with raising warning, and then fall back to normal process.